include/jemalloc/internal/jemalloc_internal_inlines_c.h

```diff
@@ -7,6 +7,12 @@
 #include "jemalloc/internal/sz.h"
 #include "jemalloc/internal/thread_event.h"
 #include "jemalloc/internal/witness.h"
+#include "jemalloc/internal/arena_externs.h"
+#include "jemalloc/internal/emap.h"
+
+#define TCACHE_IND_NONE ((unsigned)-1)
+#define TCACHE_IND_AUTOMATIC ((unsigned)-2)
+#define ARENA_IND_AUTOMATIC ((unsigned)-1)
```

Comment:
We should remove the original definition in `jemalloc.c`:
https://github.com/jemalloc/jemalloc/blob/481bbfc9906e7744716677edd49d0d6c22556a1a/src/jemalloc.c#L2281

Reply:
removed.